### ISSUE TITLE
Questions about the effect of point cloud registration

### ISSUE THREAD

**Issue by GravityTnT** [PRBonn/RAP#2]
Excellent work！I achieved satisfactory registration results using the RAP project in some angularly-defined map scenes. However, I failed to obtain excellent registration results on some datasets with complex indoor scenes. Could it be that my configurations are incorrect?
The dataset I used is 3DMatch/kitchen.

<img width="1301" height="703" alt="Image" src="https://github.com/user-attachments/assets/26afdc75-d5cf-47c7-9c8f-e819c544e536" />
Some of the parameters I set are as follows：

<img width="1505" height="310" alt="Image" src="https://github.com/user-attachments/assets/7e4b194f-ec91-4a4d-8a91-e2df9ba22927" />

**Comment by GravityTnT** [comment]
Follow up. I used 4 point clouds from the 3DSparseMatch/kitchen dataset with a similar scenario and achieved good registration results. So I'm curious as to why the above problem occurred.

<img width="841" height="558" alt="Image" src="https://github.com/user-attachments/assets/78b6799d-054c-4282-858f-3097333d4e44" />

**Comment by YuePanEdward** [comment]
Thanks for your interest in our project. You may need to turn on the option: Apply frame transform (for 3DMatch-like data with Z-axis pointing forward). Besides, I will soon update a model that would have better performance with a more flexible rotation setup.

**Comment by GravityTnT** [comment]
> Thanks for your interest in our project. You may need to turn on the option: Apply frame transform (for 3DMatch-like data with Z-axis pointing forward). Besides, I will soon update a model that would have better performance with a more flexible rotation setup.

Thank you for your prompt reply. I will continue to make attempts on the 3Dmatch dataset. Could you please tell me if this update will also include the training code? I'm really looking forward to your next steps.

**Comment by YuePanEdward** [comment]
Yes, I will also update the training code and some example training data soon.
